api_tests/providers/registrations/views/test_registration_provider_schemas.py

```diff
@@ -159,3 +177,22 @@ def test_egap_registration_schema(
         assert len(data) == 1
         assert data[0]['id'] == egap_schema._id
         assert data[0]['attributes']['name'] == egap_schema.name
+
+    def test_registration_provider_with_default_schema(
+            self,
+            app,
+            provider_with_prereg,
+            user,
+            egap_schema,
+            schema,
+            url_with_prereg,
+            osf_prereg_schema
+    ):
+        provider_with_prereg.default_schema = osf_prereg_schema
+        res = app.get(url_with_prereg, auth=user.auth)
+        assert res.status_code == 200
+        data = res.json['data']
+
+        assert len(data) == 2
+        assert osf_prereg_schema._id == data[0]['id']
```

Comment:
ordering is working because "OSF Prereg" < "Prereg Challenge", not because "OSF Prereg" is default